### PR TITLE
Cat1440

### PR DESCRIPTION
**Pull request by StefanBuerscher**
ok to test

### CODE REVIEW

**Comment by avihric** [comment]
You do not need to write `ok to test` when opening a PR. The build Is triggered automatically. Only when you want to retest your PR or for some reason, the Jenkins was down at the time you opened the PR.

**Comment by avihric** [comment]
Squash your commits please

File: catroidTest/src/org/catrobat/catroid/uitest/content/brick/VariableBricksTest.java [hunk 1]
```diff
@@ -66,6 +66,7 @@ public void tearDown() throws Exception {
 		super.tearDown();
 	}
 
+
 	public void testVariableBricks() {
```
Comment (obusher) [inline catroidTest/src/org/catrobat/catroid/uitest/content/brick/VariableBricksTest.java]: --newline;

File: catroidTest/src/org/catrobat/catroid/uitest/content/brick/VariableBricksTest.java [hunk 2]
```diff
@@ -66,6 +66,7 @@ public void tearDown() throws Exception {
 		super.tearDown();
 	}
 
+
```
Comment (obusher) [inline catroidTest/src/org/catrobat/catroid/uitest/content/brick/VariableBricksTest.java]: --newline;

**Comment by obusher** [comment]
#1297

**Pull request closed without merging**